data/remnant/remnant 2 side missions.txt

```diff
@@ -90,18 +90,18 @@ mission "Remnant: Visit to Paradise 1"
 			choice
 				`	"Interesting. How long until we arrive?"`
 					goto ourdestination
-				`	"That number seems familiar."`
+				`	"Why is that number so precise?"`
 				`	"That sounds like the speed of light."`
-			`	Taely smiles. "After all the safety factors, margins for error, and assorted other adjustments were taken into account, someone noticed that the rated value was a fraction of a meter per second above the coefficient of c, so they adjusted it down slightly so that the rated maximum velocity would match the coefficient of the speed of light in a vacuum.`
+			`	Taely smiles. "After all the safety factors, margins for error, and assorted other adjustments were taken into account, someone noticed that coefficient of the rated value was calculated to be 1.1DE79, a fraction above the coefficient of the speed of light in a vacuum, so they adjusted the rated maximum velocity down only slightly so that they would match."`
 			choice
 				`	"Thanks for the explanation."`
 					goto ourdestination
 				`	"What is a coefficient?"`
-			`	She pulls up a hologram with c=11'DE7'84A m/s on it. "Here, I will switch to base 10 to make it easier for you to understand." As she speaks, the number 299'792'458 m/s appears on the screen, and then changes to 2.99792458x10^8 m/s. "When we write out a number such as the speed of light, it is typical to break it into parts. This is generally referred to as scientific notation. This is done to facilitate comparing and working with large numbers. The first part, 2.99792458 in this case, would be called the coefficient. The second part, x10, specifies the base; and the third part, ^8 is the exponent." The display shifts again to show 'c=1.1DE784Ax16^7 m/s'. "So this would be how we typically write the speed of light when we are using it in our research." She pauses, then vanishes the display. "We will be arriving momentarily, so I should probably stop there."`
+			`	"Here, I will switch to base 10 to make it easier for you to understand." She pulls up a hologram with '299,792,458 m/s' on it. As she speaks, the number shifts to '2.99792458x10^8 m/s'. "When we write out a number such as the speed of light, it is typical to break it into parts. This is generally referred to as scientific notation. This is done to facilitate comparing and working with large numbers. The first part, 2.99792458 in this case, would be called the coefficient. The second part, x10, specifies the base; and the third part, ^8 is the exponent." The display shifts again to show '1.1DE784Ax10^7 m/s'. "So this would be how we typically write the speed of light when we are using it in our research. Remember that the value of the base is 'sixteen,' written as 10. So, 1.1DE78 is the coefficient of both the speed of light and the speed of the train, just with a different exponent." She pauses, then vanishes the display. "We will be arriving momentarily, so I should probably stop there."`
```

Comment:
```suggestion
			`	"Here, I will switch to base 10 to make it easier for you to understand." She pulls up a hologram with 299,792,458 m/s on it. As she speaks, the number shifts to 2.99792458x10^8 m/s. "When we write out a number such as the speed of light, it is typical to break it into parts. This is generally referred to as scientific notation. This is done to facilitate comparing and working with large numbers. The first part, 2.99792458 in this case, would be called the coefficient. The second part, x10, specifies the base; and the third part, ^8 is the exponent." The display shifts again to show 1.1DE784Ax10^7 m/s. "So this would be how we typically write the speed of light when we are using it in our research. Remember that the value of the base is 'sixteen,' written as 10. So, 1.1DE78 is the coefficient of both the speed of light and the speed of the train, just with a different exponent." She pauses, then vanishes the display. "We will be arriving momentarily, so I should probably stop there."`
```
The above suggestion just removes the single apostrophes around numbers.